.travis.yml

```diff
@@ -23,23 +53,34 @@ matrix:
         os: osx
 
 before_install:
-   - if [ $TRAVIS_OS_NAME == "linux" -a "$CXX" == "g++" ]; then sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test; fi
-   - if [ $TRAVIS_OS_NAME == "linux" ]; then sudo apt-get update; fi || true
+    # No addon for brew yet, have to install OSX packages this way.     
+   - if [ $TRAVIS_OS_NAME == "osx" ]; then
+        travis_wait brew install open-mpi;
+     fi
 
-install:
-   # g++-4.9
-   - if [ $TRAVIS_OS_NAME == "linux" -a "$CXX" == "g++" ]; then sudo apt-get install -qq g++-4.9; fi
-   - if [ $TRAVIS_OS_NAME == "linux" -a "$CXX" == "g++" ]; then export CXX="g++-4.9"; fi
+   # Install tool to upload code coverage reports to coveralls.io
+   - if [ "$CODECOV" == "YES" ]; then
+        pip install --user cpp-coveralls;
+        pip install --user pyyaml;
+     fi
+
```

Comment:
Container based travis configurations don't have an option to replace the default gcc version.  This was a workaround to get our builds to find gcc-4.9 first in the PATH.